build/zip.py

```diff
@@ -68,6 +68,12 @@ def main(args):
         _zip_dir(path, zip_file, archive_name)
       else:
         zip_file.write(path, archive_name)
+  if args.entitlements or args.without_entitlements:
```

Comment:
Should this be two separate if clauses? if one list exists and the other does not it will generate empty files.